simple_ado/security.py

```diff
@@ -21,22 +21,22 @@
 class ADOBranchPermission(enum.IntEnum):
```

Comment:
These changes should be undone.

Reply:
Believe these came from running the style checks.